### PR TITLE
Rename variable in examples for clarity

### PR DESCRIPTION
**Pull request by akarve**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ provider "aws" {
 }
 
 locals {
-  name           = ""
+  name                = ""
   // You receive the build_file CloudFormation Template from your Quilt account
```
Comment (dimaryaz) [inline README.md:23]: Should rename it in the comment, too?

Comment (akarve) [inline README.md:23]:
```suggestion
  // Place a local copy of your CloudFormation YAML Template at build_file_path
```

File: README.md [hunk 2]
```diff
@@ -19,11 +19,11 @@ provider "aws" {
 }
 
 locals {
-  name           = ""
-  // You receive the build_file CloudFormation Template from your Quilt account
+  name                = ""
+  // Place a local copy of your CloudFormation YAML Template at build_file_path
   // manager and check it into git
```
Comment (akarve) [inline README.md:24]:
```suggestion
  // and check it into git
```

File: README.md (hunk 1 above)
Comment (akarve) [reply README.md]: Sure.

**Review by dimaryaz — APPROVED** [review]

**Pull request merged**